repos/fdbt-site/tests/pages/api/salesConfirmation.test.ts

```diff
@@ -220,66 +215,4 @@ describe('salesOfferPackages', () => {
 
         expect(getGeoZoneTicketJsonSpy).toBeCalledWith(req, res);
     });
-
-    it('creates a group definition for a group ticket and adds to user json object', async () => {
```

Comment:
Is it worth creating a unit test, to ensure it calls the correct methods etc or DB calls for a single passenger type and for a group passenger type?